doc/code/qml_pauli.rst

```diff
@@ -14,7 +14,7 @@ for Pauli-word partitioning functionality used in measurement optimization.
     :no-heading:
     :no-main-docstr:
     :no-inherited-members:
-    :skip: lie_closure, structure_constants, center
+    :skip:
```

Comment:
I'm still seeing `lie_closure`, `structure_constants`, and `center` in the [docs for qml.pauli](https://xanaduai-pennylane--6935.com.readthedocs.build/en/6935/code/qml_pauli.html). Is this intended?

Reply:
Ah, it's staying in here and deprecated until it's removed. That makes sense.